packages/core/contracts/zodiac/OptimisticOracleModule.sol

```diff
@@ -0,0 +1,232 @@
+// SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
is the intention to write unit tests for this? even if you mock the external contracts it's properly a good idea to do so to validate that the right functions are called! We have a bunch of tooling to do this with [smock](http://smock.readthedocs.io)

Reply:
Yep, I'm working on tests right now.

Reply:
Digging around for smock specifically, can you show where we use that so I can see examples?